web/src/components/organisms/Settings/Workspace/index.tsx

```diff
@@ -0,0 +1,195 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import List from "@reearth-cms/components/atoms/List";
+import React, { useCallback, useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+import useHooks from "./hooks";
+
+const Workspace: React.FC = () => {
+  const { workspaceId } = useParams();
+
+  const {
+    me,
+    currentWorkspace,
+    searchedUser,
+    changeSearchedUser,
+    handleWorkspaceDelete,
+    handleUserSearch,
+    handleNameUpdate,
+    handleMemberAddToWorkspace,
+    handleMemberOfWorkspaceUpdate,
+    handleMemberRemoveFromWorkspace,
+  } = useHooks({ workspaceId });
+
+  const [owner, setOwner] = useState(false);
+  const [memberName, setMemberName] = useState("");
+  const [workspaceName, setWorkspaceName] = useState("");
+
+  const navigate = useNavigate();
+  const members = currentWorkspace?.members;
+
+  const handleMemberNameChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      setMemberName?.(e.currentTarget.value);
+      handleUserSearch(e.currentTarget.value);
+    },
+    [setMemberName, handleUserSearch]
+  );
+
+  const handleMemberAdd = useCallback(() => {
+    if (!searchedUser) return;
+    handleMemberAddToWorkspace([searchedUser.id]);
+    setMemberName("");
+    changeSearchedUser(undefined);
+  }, [searchedUser, handleMemberAddToWorkspace, changeSearchedUser]);
+
+  const handleWorkspaceDeleteAction = useCallback(async () => {
```

Comment:
Sorry for all the small asks, but can you do 
```suggestion
  const handleWorkspaceDeletion = useCallback(async () => {
```